typescript/hello-world/src/app.ts

```diff
@@ -6,11 +6,11 @@ import express from 'express';
 // Shim for mattermost-redux global fetch access
 global.fetch = require('node-fetch');
 
-import {AppBinding, AppCallRequest, AppCallResponse, AppForm, AppManifest} from 'mattermost-redux/types/apps';
-import {Post} from 'mattermost-redux/types/posts';
-import {Channel} from 'mattermost-redux/types/channels';
+import {AppBinding, AppCallRequest, AppCallResponse, AppForm, AppManifest} from '@mattermost/types/lib/apps';
+import {Post} from '@mattermost/types/lib/posts';
+import {Channel} from '@mattermost/types/lib/channels';
```

Comment:
I've changed `tsconfig.json` on this branch so that we can import these as desired:

```suggestion
import {AppBinding, AppCallRequest, AppCallResponse, AppForm, AppManifest} from '@mattermost/types/apps';
import {Post} from '@mattermost/types/posts';
import {Channel} from '@mattermost/types/channels';
```

Reply:
I can't get the import path working. `@mattermost/types/posts` given me an error
![Screenshot from 2022-11-16 18-29-33](https://user-images.githubusercontent.com/16541325/202251569-4f5ce918-d201-441d-8ed8-29c4330f7189.png)

Is there some additional configuration needed?

Reply:
You may need to reload your vscode window via `Cmd P` `Reload Window` before this works, in case you pulled and kept the same vscode session open. I suppose I'm fine with importing from `@mattermost/types/lib` for the time being. Eventually when `@mattermost/apps` is available, we'll want to `exclude` `@mattermost/types/apps`. Hopefully that will work as intended.

Reply:
I still can't make it work. Let's stick with `@mattermost/types/lib`.